tls/autoregenerate-after-expiry/autoregenerate-after-expiry.md

```diff
@@ -39,17 +40,23 @@
   - [Items That DO Meet the Requirement (0)](#Items-That-DO-Meet-the-Requirement-0)
 
 
+## How to meet the requirement
 Acknowledging that a cert/key pair or CA bundle can auto-regenerate after it expires offline means
 that if the cluster is shut down until the certificate expires, when the machines are restarted
 the cluster will automatically create new cert/key pairs or update CA bundles as required without human
 intervention.
-To assert that a particular cert/key pair or CA bundle can do this, add the "certificates.openshift.io/auto-regenerate-after-offline-expiry" annotation to the secret or configmap and 
-setting the value of the annotation a github link to the PR adding the annotation.
-This assertion also means that you have
+
+To assert that a particular cert/key pair or CA bundle can do this, add the annotation to the secret or configmap.
+```yaml
+  annotations:
+    certificates.openshift.io/auto-regenerate-after-offline-expiry: https//github.com/link/to/pr/adding/annotation, "quote escaped formatted name of e2e test that ensures the PKI artifact functions properly"
+```
+
+This assertion means that you have
 1. Manually tested that this works or seen someone else manually test that this works.  AND
-2. Written an automated e2e job that your team has an alert for and is a blocking GA criteria, and/or
+2. Written an automated e2e test to ensure this PKI artifact is function that is a blocking GA criteria, and/or
       QE has required test every release that ensures the functionality works every release.
-Links should be provided in the PR adding the annotation.
+If you have not done this, you should not merge the annotation.
```

Comment:
```suggestion
If you have not done this, you should not merge the PR setting this annotation.
```

Reply:
@vrutkovs based on William's comment, I'm going to merge this to help us communicate where we're at with a perma-link.

I'm completely fine adding this extra verbiage, but I need to merge with green verifies.